inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -509,4 +520,36 @@ public function display_wrong_license_notice() {
 			]
 		);
 	}
+
+	/**
+	 * Display a notice on table missing.
+	 *
+	 * @return void
+	 */
+	public function display_no_table_notice() {
+		if ( ! $this->can_display_notice() ) {
+			return;
+		}
+
+		if ( $this->used_css->exists() ) {
+			return;
+		}
+
+		$main_message = __( 'We detected missing database table related to the Remove Unused CSS feature.', 'rocket' );
```

Comment:
We can use the text provided here https://github.com/wp-media/wp-rocket/issues/3960#issuecomment-1180433721